packages/xo-server-usage-report/src/index.js

```diff
@@ -136,6 +136,10 @@ Handlebars.registerHelper('math', function (lvalue, operator, rvalue, options) {
 
 Handlebars.registerHelper('shortUUID', shortUuid)
 
+Handlebars.registerHelper('formatAddresses', addresses =>
+  addresses.length === 0 ? 'No IP record' : addresses.join(', ')
```

Comment:
1. zero is (usually) plural in English
2. *record* is IMHO unclear here

```suggestion
  addresses.length === 0 ? 'No IP addresses' : addresses.join(', ')
```

Reply:
I used the same denomination than the UI

![image](https://user-images.githubusercontent.com/21563339/112615239-85d08680-8e22-11eb-9fe0-6cc11899380f.png)